src/App.js

```diff
@@ -1,7 +1,33 @@
 import "./App.css";
+import Header from "./components/Header";
+import { Route, Routes } from "react-router-dom";
+import Nav from "./components/Nav";
+import Articles from "./components/Articles";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <p>
+              <Header />
```

Comment:
What is the purpose of this component? Is it supposed to be displayed as well as other components and so should be out side of a Route? At the moment it just contains a link to itself.